quantum/nodes/hadamard-gate/hadamard-gate.html

```diff
@@ -0,0 +1,36 @@
+<script type="text/javascript">
+    RED.nodes.registerType("hadamard-gate", {
+      category: "quantum",
+      color: "#FA4D56",
+      defaults: {
+        name: { value: "H" },
+      },
+      inputs: 1,
+      outputs: 1,
+      icon: "font-awesome/fa-header",
+      align: 'right',
+      paletteLabel: "H",
+      label: function () {
+        return this.name || "H";
+      },
+      inputLabels: function(index) {
+          return "qubit " + index;
+      },
+      outputLabels: "H",
```

Comment:
```suggestion
      paletteLabel: "hadamard",
      label: function () {
        return this.name || "hadamard";
      },
      inputLabels: function(index) {
          return "qubit " + index;
      },
      outputLabels: "hadamard",
```
We should have the full name of the gate here, not just a "H".